futurex_openedx_extensions/dashboard/views.py

```diff
@@ -219,3 +226,24 @@ def get(self, request, *args, **kwargs):  # pylint: disable=no-self-use
         return JsonResponse({
             'version': futurex_openedx_extensions.__version__,
         })
+
+
+class AccessibleTenantsInfoView(APIView):
+    """View to get the list of accessible tenants"""
+    permission_classes = []
```

Comment:
Please add `Token` authentication and limit to superusers.

Reply:
will do 👍🏼

Reply:
@OmarIthawi , I added a rate limit for anonymous access. The API is accessible as anonymous or system-staff. No rate limit on system-staff users. Do we still need an API Key or Token? I guess not